src/logger/loggerApi.c

```diff
@@ -1118,11 +1123,14 @@ static void sendTimerConfig(Serial *serial, size_t startIndex, size_t endIndex)
         TimerConfig *cfg = &(getWorkingLoggerConfig()->TimerConfigs[i]);
         json_objStartInt(serial, i);
         json_channelConfig(serial, &(cfg->cfg), 1);
-        json_uint(serial, "st", cfg->slowTimerEnabled, 1);
+         /* DEPRECATED.  Only here for compatibility */
+        json_uint(serial, "st", 0, 1);
```

Comment:
we should check if the app fails if the value is missing... if it doesn't fail, we should remove it.